packages/mobile/src/screens/edit-track-screen/components/RemixTrackPill.tsx

```diff
@@ -69,10 +69,13 @@ export const RemixTrackPill = (props: RemixTrackPillProps) => {
         style={{ height: 20, width: 20 }}
         borderRadius='xs'
       />
-      <Flex direction='row' alignItems='center'>
+      <Flex direction='row' alignItems='center' style={{ flexShrink: 1 }}>
```

Comment:
nit: can just do `row` instead of `direction='row'` now

Reply:
very true